examples/angular/README.md

```diff
@@ -0,0 +1,41 @@
+# AngularExample
```

Comment:
This should be aligned with existing readmes: https://github.com/contentful/apps/blob/master/examples/typescript/README.md